addons/web_tour/static/src/js/tip.js

```diff
@@ -205,7 +205,7 @@ var Tip = Widget.extend({
             // important to only remove/add the class when necessary to not
             // notify a DOM mutation which could retrigger this function.
             const $oldLocation = this.$el.parent();
-            if (!$location.is($oldLocation)) {
```

Comment:
Is this one really necessary? Can you explain why?

Reply:
Actually this fix is related to https://www.youtube.com/watch?v=StFyJIR6OAU&feature=youtu.be

Generally when tip bubble and its info is in viewport(not in control panel) then it is rendering besides the element itself. but when the tip info is about the control buttons, it renders as a body element directly. 
Now what happened, when tip bubble is rendered for the control button (i.e. Test button) then it is rendered beside the form sheet, not as a body element. but when you hover on that bubble then it will glitch for the first time as its add the info element besides the sheet and then it renders in the body directly.

so with this condition, we are restricting to reattach the info div at the same div where the bubble div it self.